test/data/json/create-tiny.js

```diff
@@ -0,0 +1,73 @@
+/// <reference types="node" />
+
+'use strict';
+
+import {sliceJSONChrom, writeJSONChrom, sliceAndWriteJSONChrom } from '../../../src/chromUtils'
```

Comment:
The import statement should be like

```js
import {sliceJSONChrom, writeJSONChrom, sliceAndWriteJSONChrom } from '../../../'
```

but it's not going to work either because the funtion it's not exported from index